src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -234,7 +235,9 @@
     :ellipsize-mode      :tail
     :weight              :semi-bold
     :size                :heading-1
-    :style               {:margin-top 56}}
+    :style               {:margin-top (+ scroll-page.style/picture-radius
```

Comment:
FYI: I parameterized certain style values. I think this practice of magical values is one of the sources of constant bugs in the UI. The question I ask myself to decide if we need vars is: *if somebody changes the style value, will they remember to change this magical number?* If the answer is no, then ideally, vars/constants should be used.